openfisca_us/variables/gov/states/in/tax/income/deductions/in_military_service_deduction.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_us.model_api import *
+
+
+class in_military_service_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN military service deduction"
+    definition_period = YEAR
+    unit = USD
+    reference = "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-2-4"  # (a)(1)
+
+    def formula(tax_unit, period, parameters):
```

Comment:
I think we can unify this with `in_person_military_service` and make it tax-unit-level, for consistency with exemptions and to save memory. To do this, start with `person = tax_unit.members` and then port everything from the other formula, and `return tax_unit.sum(person_deduction)`.

But for future reference, `formula = sum_of_variables(["in_person_military_service_deduction"])` would have worked here.